test/functional/lcow_bench_test.go

```diff
@@ -0,0 +1,101 @@
+//go:build windows && functional
+// +build windows,functional
+
+package functional
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Microsoft/hcsshim/osversion"
+
+	"github.com/Microsoft/hcsshim/test/internal/require"
+	"github.com/Microsoft/hcsshim/test/internal/uvm"
+)
+
+func BenchmarkLCOW_UVM_Create(b *testing.B) {
+	requireFeatures(b, featureLCOW)
+	require.Build(b, osversion.RS5)
+
+	ctx := context.Background()
+
+	b.StopTimer()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		opts := defaultLCOWOptions(b, b.Name())
```

Comment:
Does each iteration count as a "subtest" for the benchmark? If no, wouldn't b.Name() not be unique?

Reply:
So its a little weird, but this is based on how [the docs tell you to do it](https://pkg.go.dev/testing#hdr-Benchmarks), a benchmark run of `BenchmarkLCOW_UVM_Create` will include as many iterations (`b.N`) the target `-benchtime` criteria is met (either a set number, eg. `-benchtime 1000x` or duration, eg. `-benchtime 3s`)
All those iterations are accumulated under one timer, which can then be repeated via `-count`

So if we with 
'-test.count=2'  '-test.run=^#' '-test.benchmem'  '-test.bench=.' '-test.benchtime=2x'
The output would look something like (I dont know why go append the `12` here):
```
BenchmarkLCOW_UVM_Create-12                      	       2	  84758700 ns/op	   47800 B/op	     363 allocs/op
BenchmarkLCOW_UVM_Create-12                      	       2	  81733600 ns/op	   42584 B/op	     348 allocs/op
BenchmarkLCOW_UVM_Create-12                       	       2	  83118950 ns/op	   43904 B/op	     385 allocs/op
```
benchstat then aggregates benchmarks with the same name, so we want the non-uniqueness:
```
LCOW_UVM_Create-12               83.2ms ± 2%
```

Reply:
how does creating a new LCOW work if the names are the same?

Reply:
the `cleanup()` call on line 33 deletes the uvm, so the name can be reused